examples/community/README.md

```diff
@@ -97,7 +97,7 @@ from diffusers import DiffusionPipeline
 import torch
 
 pipe = DiffusionPipeline.from_pretrained(
-    "CompVis/stable-diffusion-v1-4",
+    "runwayml/stable-diffusion-v1-5",
```

Comment:
Let's maybe revert this as well as the community pipeline author should decide here